### ISSUE TITLE
[Bug]: TimescaleDB not releasing space from disk after compression

### ISSUE THREAD

**Issue by FireHelmet** [timescale/timescaledb#5582]
### What type of bug is this?

Other

### What subsystems and features are affected?

Compression, Other

### What happened?

I encounter the same issue as #3289 , My TimescaleDB instance is running in a container and data are stored in a docker volume.

My database size is 70GB,

```shell
zabbix=> SELECT pg_size_pretty( pg_database_size('zabbix'));
 pg_size_pretty
----------------
 70 GB
``` 

The compression jobs are working without errors,

```shell
zabbix=> SELECT hypertable_name, job_status, total_runs, total_successes, total_failures FROM timescaledb_information.job_stats;
 hypertable_name | job_status | total_runs | total_successes | total_failures
-----------------+------------+------------+-----------------+----------------
 history         | Scheduled  |        105 |             105 |              0
 history_log     | Scheduled  |        105 |             105 |              0
 history_str     | Scheduled  |        105 |             105 |              0
 history_text    | Scheduled  |        105 |             105 |              0
 history_uint    | Scheduled  |        105 |             105 |              0
 trends          | Scheduled  |        105 |             105 |              0
 trends_uint     | Scheduled  |        105 |             105 |              0
``` 

The chunks are compressed (biggest table),

```shell
zabbix=> SELECT total_chunks, number_compressed_chunks, pg_size_pretty(before_compression_total_bytes) as "before compression",pg_size_pretty(after_compression_total_bytes) as "after compression" FROM hypertable_compression_stats('history_uint');
 total_chunks | number_compressed_chunks | before compression | after compression
--------------+--------------------------+--------------------+-------------------
           91 |                       82 | 69 GB              | 4274 MB
(1 row)
``` 

But the free space isn't released,

```shell
Filesystem                Size      Used Available Use% Mounted on
/dev/sdc1               119.2G     72.7G     46.5G  61% /var/lib/postgresql/data
``` 

```shell
81ad4639bc2f:/var/lib/postgresql/data# du -h
9.8M    ./base/1
8.2M    ./base/13768
9.8M    ./base/13769
10.5M   ./base/16384
71.1G   ./base/18412
22.9M   ./base/23253
0       ./base/pgsql_tmp
71.1G   ./base
``` 

Should I launch a process to reclaim space ? What are the options to solve it ?

Many thanks in advance for your support,

### TimescaleDB version affected

2.10.1

### PostgreSQL version used

14.7

### What operating system did you use?

Alpine Linux v3.17

### What installation method did you use?

Docker

### What platform did you run on?

Microsoft Azure Cloud

### How can we reproduce the bug?

```bash
I don't know
```

**Comment by FireHelmet** [comment]
@akuzm ,

executed as zabbix user:

```shell
zabbix=> select 18412::oid::regclass;
 regclass
----------
 18412
(1 row)
```

**Comment by akuzm** [comment]
Ah, OK, 18412 is the zabbix database. Can you drill down to `./base/18412` with `du`, to see which tables are taking up space? You can run the same thing with `::regclass` and `select from _timescaledb_internal.chunks` for the biggest ones, to check what they are.

**Comment by FireHelmet** [comment]
@akuzm 

```shell
zabbix=> SELECT * , pg_size_pretty(total_bytes) as "total_bytes" FROM chunks_detailed_size('history');
     chunk_schema      |     chunk_name     | table_bytes | index_bytes | toast_bytes | total_bytes | node_name | total_bytes
-----------------------+--------------------+-------------+-------------+-------------+-------------+-----------+-------------
 _timescaledb_internal | _hyper_1_199_chunk |       40960 |       32768 |      868352 |      942080 |           | 920 kB
 _timescaledb_internal | _hyper_1_670_chunk |  1155055616 |  1069719552 |           0 |  2224775168 |           | 2122 MB
 _timescaledb_internal | _hyper_1_686_chunk |   794968064 |   729579520 |           0 |  1524547584 |           | 1454 MB
 _timescaledb_internal | _hyper_1_662_chunk |  1144381440 |  1066098688 |           0 |  2210480128 |           | 2108 MB
 _timescaledb_internal | _hyper_1_211_chunk |       73728 |       32768 |     1187840 |     1294336 |           | 1264 kB
 _timescaledb_internal | _hyper_1_242_chunk |       65536 |       49152 |     2260992 |     2375680 |           | 2320 kB
 _timescaledb_internal | _hyper_1_254_chunk |       81920 |       49152 |     3014656 |     3145728 |           | 3072 kB
 _timescaledb_internal | _hyper_1_259_chunk |       90112 |       57344 |     3579904 |     3727360 |           | 3640 kB
 _timescaledb_internal | _hyper_1_284_chunk |      122880 |       57344 |     4669440 |     4849664 |           | 4736 kB
 _timescaledb_internal | _hyper_1_678_chunk |  1166647296 |  1089388544 |           0 |  2256035840 |           | 2152 MB
 _timescaledb_internal | _hyper_1_474_chunk |      851968 |      155648 |    23552000 |    24559616 |           | 23 MB
 _timescaledb_internal | _hyper_1_75_chunk  |       40960 |       32768 |      581632 |      655360 |           | 640 kB
 _timescaledb_internal | _hyper_1_218_chunk |       65536 |       49152 |     2056192 |     2170880 |           | 2120 kB
 _timescaledb_internal | _hyper_1_224_chunk |       65536 |       49152 |     2056192 |     2170880 |           | 2120 kB
 _timescaledb_internal | _hyper_1_230_chunk |       65536 |       49152 |     2056192 |     2170880 |           | 2120 kB
 _timescaledb_internal | _hyper_1_236_chunk |       65536 |       49152 |     2138112 |     2252800 |           | 2200 kB
 _timescaledb_internal | _hyper_1_248_chunk |       73728 |       49152 |     2482176 |     2605056 |           | 2544 kB
 _timescaledb_internal | _hyper_1_265_chunk |      106496 |       57344 |     4472832 |     4636672 |           | 4528 kB
 _timescaledb_internal | _hyper_1_278_chunk |       90112 |       57344 |     4079616 |     4227072 |           | 4128 kB
 _timescaledb_internal | _hyper_1_290_chunk |      131072 |       65536 |     6307840 |     6504448 |           | 6352 kB
 _timescaledb_internal | _hyper_1_81_chunk  |       40960 |       32768 |      581632 |      655360 |           | 640 kB
 _timescaledb_internal | _hyper_1_87_chunk  |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_93_chunk  |       40960 |       32768 |      581632 |      655360 |           | 640 kB
 _timescaledb_internal | _hyper_1_99_chunk  |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_105_chunk |       49152 |       32768 |      581632 |      663552 |           | 648 kB
 _timescaledb_internal | _hyper_1_193_chunk |       40960 |       32768 |      851968 |      925696 |           | 904 kB
 _timescaledb_internal | _hyper_1_304_chunk |      106496 |       57344 |     4841472 |     5005312 |           | 4888 kB
 _timescaledb_internal | _hyper_1_316_chunk |      360448 |       49152 |       57344 |      466944 |           | 456 kB
 _timescaledb_internal | _hyper_1_328_chunk |      114688 |       57344 |     4800512 |     4972544 |           | 4856 kB
 _timescaledb_internal | _hyper_1_348_chunk |       81920 |       49152 |     4104192 |     4235264 |           | 4136 kB
 _timescaledb_internal | _hyper_1_113_chunk |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_205_chunk |       49152 |       32768 |      851968 |      933888 |           | 912 kB
 _timescaledb_internal | _hyper_1_271_chunk |      106496 |       57344 |     4464640 |     4628480 |           | 4520 kB
 _timescaledb_internal | _hyper_1_298_chunk |      106496 |       57344 |     4816896 |     4980736 |           | 4864 kB
 _timescaledb_internal | _hyper_1_310_chunk |      106496 |       57344 |     4825088 |     4988928 |           | 4872 kB
 _timescaledb_internal | _hyper_1_324_chunk |       65536 |       49152 |     2777088 |     2891776 |           | 2824 kB
 _timescaledb_internal | _hyper_1_342_chunk |      139264 |       65536 |     4956160 |     5160960 |           | 5040 kB
 _timescaledb_internal | _hyper_1_143_chunk |       49152 |       32768 |      581632 |      663552 |           | 648 kB
 _timescaledb_internal | _hyper_1_149_chunk |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_157_chunk |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_119_chunk |       40960 |       32768 |      581632 |      655360 |           | 640 kB
 _timescaledb_internal | _hyper_1_125_chunk |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_131_chunk |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_137_chunk |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_163_chunk |       40960 |       32768 |      573440 |      647168 |           | 632 kB
 _timescaledb_internal | _hyper_1_169_chunk |       49152 |       32768 |      622592 |      704512 |           | 688 kB
 _timescaledb_internal | _hyper_1_175_chunk |       40960 |       32768 |      729088 |      802816 |           | 784 kB
 _timescaledb_internal | _hyper_1_181_chunk |       40960 |       32768 |      851968 |      925696 |           | 904 kB
 _timescaledb_internal | _hyper_1_187_chunk |       40960 |       32768 |      851968 |      925696 |           | 904 kB
 _timescaledb_internal | _hyper_1_334_chunk |      106496 |       57344 |     4808704 |     4972544 |           | 4856 kB
 _timescaledb_internal | _hyper_1_353_chunk |       57344 |       32768 |     1900544 |     1990656 |           | 1944 kB
 _timescaledb_internal | _hyper_1_360_chunk |       65536 |       32768 |     1867776 |     1966080 |           | 1920 kB
 _timescaledb_internal | _hyper_1_366_chunk |      131072 |       49152 |     2818048 |     2998272 |           | 2928 kB
 _timescaledb_internal | _hyper_1_372_chunk |      114688 |       57344 |     4071424 |     4243456 |           | 4144 kB
 _timescaledb_internal | _hyper_1_378_chunk |      114688 |       57344 |     4038656 |     4210688 |           | 4112 kB
 _timescaledb_internal | _hyper_1_384_chunk |      114688 |       57344 |     4046848 |     4218880 |           | 4120 kB
 _timescaledb_internal | _hyper_1_390_chunk |      122880 |       65536 |     4358144 |     4546560 |           | 4440 kB
 _timescaledb_internal | _hyper_1_606_chunk |     4055040 |      827392 |   182599680 |   187482112 |           | 179 MB
 _timescaledb_internal | _hyper_1_490_chunk |     1949696 |      212992 |    31997952 |    34160640 |           | 33 MB
 _timescaledb_internal | _hyper_1_498_chunk |     2662400 |      466944 |    86327296 |    89456640 |           | 85 MB
 _timescaledb_internal | _hyper_1_507_chunk |     3670016 |      729088 |   156262400 |   160661504 |           | 153 MB
 _timescaledb_internal | _hyper_1_558_chunk |     7348224 |      483328 |    46292992 |    54124544 |           | 52 MB
 _timescaledb_internal | _hyper_1_630_chunk |  1092771840 |  1040416768 |           0 |  2133188608 |           | 2034 MB
 _timescaledb_internal | _hyper_1_414_chunk |      147456 |       65536 |     5275648 |     5488640 |           | 5360 kB
 _timescaledb_internal | _hyper_1_420_chunk |      172032 |       73728 |     6217728 |     6463488 |           | 6312 kB
 _timescaledb_internal | _hyper_1_598_chunk |     4218880 |      835584 |   184565760 |   189620224 |           | 181 MB
 _timescaledb_internal | _hyper_1_614_chunk |     4210688 |      835584 |   182239232 |   187285504 |           | 179 MB
 _timescaledb_internal | _hyper_1_622_chunk |     4202496 |      835584 |   182984704 |   188022784 |           | 179 MB
 _timescaledb_internal | _hyper_1_395_chunk |      122880 |       65536 |     4702208 |     4890624 |           | 4776 kB
 _timescaledb_internal | _hyper_1_401_chunk |      122880 |       65536 |     4702208 |     4890624 |           | 4776 kB
 _timescaledb_internal | _hyper_1_482_chunk |      884736 |      180224 |    31555584 |    32620544 |           | 31 MB
 _timescaledb_internal | _hyper_1_638_chunk |  1092878336 |  1036926976 |           0 |  2129805312 |           | 2031 MB
 _timescaledb_internal | _hyper_1_426_chunk |      204800 |       81920 |     6840320 |     7127040 |           | 6960 kB
 _timescaledb_internal | _hyper_1_432_chunk |      278528 |       81920 |     8052736 |     8413184 |           | 8216 kB
 _timescaledb_internal | _hyper_1_439_chunk |      262144 |       81920 |    10141696 |    10485760 |           | 10 MB
 _timescaledb_internal | _hyper_1_446_chunk |      262144 |       81920 |    10117120 |    10461184 |           | 10216 kB
 _timescaledb_internal | _hyper_1_468_chunk |      712704 |      131072 |    18792448 |    19636224 |           | 19 MB
 _timescaledb_internal | _hyper_1_565_chunk |     1908736 |      458752 |    90431488 |    92798976 |           | 89 MB
 _timescaledb_internal | _hyper_1_574_chunk |      229376 |       81920 |    10321920 |    10633216 |           | 10 MB
 _timescaledb_internal | _hyper_1_581_chunk |      360448 |       90112 |    10657792 |    11108352 |           | 11 MB
 _timescaledb_internal | _hyper_1_646_chunk |  1110671360 |  1049673728 |           0 |  2160345088 |           | 2060 MB
 _timescaledb_internal | _hyper_1_654_chunk |  1136926720 |  1058021376 |           0 |  2194948096 |           | 2093 MB
 _timescaledb_internal | _hyper_1_408_chunk |      131072 |       65536 |     5005312 |     5201920 |           | 5080 kB
 _timescaledb_internal | _hyper_1_453_chunk |      458752 |      106496 |    12328960 |    12894208 |           | 12 MB
 _timescaledb_internal | _hyper_1_460_chunk |      630784 |      122880 |    17956864 |    18710528 |           | 18 MB
 _timescaledb_internal | _hyper_1_515_chunk |     3956736 |      761856 |   166797312 |   171515904 |           | 164 MB
 _timescaledb_internal | _hyper_1_532_chunk |     3981312 |      811008 |   179535872 |   184328192 |           | 176 MB
 _timescaledb_internal | _hyper_1_540_chunk |     4268032 |      868352 |   191864832 |   197001216 |           | 188 MB
 _timescaledb_internal | _hyper_1_550_chunk |     2007040 |      475136 |   100810752 |   103292928 |           | 99 MB
 _timescaledb_internal | _hyper_1_590_chunk |     2064384 |      475136 |    95756288 |    98295808 |           | 94 MB
 _timescaledb_internal | _hyper_1_524_chunk |     3817472 |      770048 |   169091072 |   173678592 |           | 166 MB
(91 rows)

``` 

```shell
zabbix=> SELECT * , pg_size_pretty(total_bytes) as "total_bytes" FROM chunks_detailed_size('history_uint');
     chunk_schema      |     chunk_name     | table_bytes | index_bytes | toast_bytes | total_bytes | node_name | total_bytes
-----------------------+--------------------+-------------+-------------+-------------+-------------+-----------+-------------
 _timescaledb_internal | _hyper_2_661_chunk |  2860032000 |  2773008384 |           0 |  5633040384 |           | 5372 MB
 _timescaledb_internal | _hyper_2_677_chunk |  2868715520 |  2755371008 |           0 |  5624086528 |           | 5364 MB
 _timescaledb_internal | _hyper_2_685_chunk |  1953792000 |  1832828928 |           0 |  3786620928 |           | 3611 MB
 _timescaledb_internal | _hyper_2_76_chunk  |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_194_chunk |      663552 |      147456 |     9437184 |    10248192 |           | 10008 kB
 _timescaledb_internal | _hyper_2_200_chunk |     1540096 |      172032 |     6676480 |     8388608 |           | 8192 kB
 _timescaledb_internal | _hyper_2_217_chunk |       90112 |       57344 |     3186688 |     3334144 |           | 3256 kB
 _timescaledb_internal | _hyper_2_223_chunk |       90112 |       57344 |     3170304 |     3317760 |           | 3240 kB
 _timescaledb_internal | _hyper_2_229_chunk |       90112 |       57344 |     3178496 |     3325952 |           | 3248 kB
 _timescaledb_internal | _hyper_2_241_chunk |      122880 |       65536 |     4513792 |     4702208 |           | 4592 kB
 _timescaledb_internal | _hyper_2_247_chunk |      131072 |       65536 |     5005312 |     5201920 |           | 5080 kB
 _timescaledb_internal | _hyper_2_260_chunk |      163840 |       73728 |     6275072 |     6512640 |           | 6360 kB
 _timescaledb_internal | _hyper_2_272_chunk |      172032 |       73728 |     6774784 |     7020544 |           | 6856 kB
 _timescaledb_internal | _hyper_2_309_chunk |      811008 |      229376 |    25485312 |    26525696 |           | 25 MB
 _timescaledb_internal | _hyper_2_669_chunk |  2860572672 |  2767683584 |           0 |  5628256256 |           | 5368 MB
 _timescaledb_internal | _hyper_2_212_chunk |       65536 |       49152 |     1343488 |     1458176 |           | 1424 kB
 _timescaledb_internal | _hyper_2_235_chunk |      122880 |       57344 |     3604480 |     3784704 |           | 3696 kB
 _timescaledb_internal | _hyper_2_253_chunk |      139264 |       65536 |     5464064 |     5668864 |           | 5536 kB
 _timescaledb_internal | _hyper_2_289_chunk |      204800 |       81920 |     7651328 |     7938048 |           | 7752 kB
 _timescaledb_internal | _hyper_2_475_chunk |     3244032 |      663552 |    74792960 |    78700544 |           | 75 MB
 _timescaledb_internal | _hyper_2_82_chunk  |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_88_chunk  |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_94_chunk  |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_100_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_106_chunk |       40960 |       32768 |       98304 |      172032 |           | 168 kB
 _timescaledb_internal | _hyper_2_266_chunk |      172032 |       73728 |     6774784 |     7020544 |           | 6856 kB
 _timescaledb_internal | _hyper_2_333_chunk |      696320 |      196608 |    21512192 |    22405120 |           | 21 MB
 _timescaledb_internal | _hyper_2_341_chunk |      802816 |      204800 |    21659648 |    22667264 |           | 22 MB
 _timescaledb_internal | _hyper_2_347_chunk |      450560 |      131072 |    14655488 |    15237120 |           | 15 MB
 _timescaledb_internal | _hyper_2_206_chunk |       49152 |       32768 |      663552 |      745472 |           | 728 kB
 _timescaledb_internal | _hyper_2_277_chunk |      172032 |       73728 |     6750208 |     6995968 |           | 6832 kB
 _timescaledb_internal | _hyper_2_283_chunk |      188416 |       81920 |     6938624 |     7208960 |           | 7040 kB
 _timescaledb_internal | _hyper_2_297_chunk |      532480 |      155648 |    13656064 |    14344192 |           | 14 MB
 _timescaledb_internal | _hyper_2_303_chunk |      835584 |      229376 |    25706496 |    26771456 |           | 26 MB
 _timescaledb_internal | _hyper_2_315_chunk |     2457600 |      131072 |      114688 |     2703360 |           | 2640 kB
 _timescaledb_internal | _hyper_2_325_chunk |      425984 |      131072 |    13639680 |    14196736 |           | 14 MB
 _timescaledb_internal | _hyper_2_327_chunk |      843776 |      221184 |    25296896 |    26361856 |           | 25 MB
 _timescaledb_internal | _hyper_2_144_chunk |       40960 |       32768 |       98304 |      172032 |           | 168 kB
 _timescaledb_internal | _hyper_2_150_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_158_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_114_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_126_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_132_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_138_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_120_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_164_chunk |        8192 |       32768 |       90112 |      131072 |           | 128 kB
 _timescaledb_internal | _hyper_2_170_chunk |      999424 |       81920 |      507904 |     1589248 |           | 1552 kB
 _timescaledb_internal | _hyper_2_176_chunk |     2490368 |      139264 |     3710976 |     6340608 |           | 6192 kB
 _timescaledb_internal | _hyper_2_182_chunk |      663552 |      147456 |     9396224 |    10207232 |           | 9968 kB
 _timescaledb_internal | _hyper_2_188_chunk |      630784 |      147456 |     9281536 |    10059776 |           | 9824 kB
 _timescaledb_internal | _hyper_2_413_chunk |      745472 |      204800 |    23642112 |    24592384 |           | 23 MB
 _timescaledb_internal | _hyper_2_354_chunk |       73728 |       49152 |     1835008 |     1957888 |           | 1912 kB
 _timescaledb_internal | _hyper_2_359_chunk |       73728 |       49152 |     1810432 |     1933312 |           | 1888 kB
 _timescaledb_internal | _hyper_2_365_chunk |      475136 |      122880 |    12238848 |    12836864 |           | 12 MB
 _timescaledb_internal | _hyper_2_371_chunk |      729088 |      196608 |    22175744 |    23101440 |           | 22 MB
 _timescaledb_internal | _hyper_2_377_chunk |      712704 |      196608 |    22167552 |    23076864 |           | 22 MB
 _timescaledb_internal | _hyper_2_383_chunk |      704512 |      196608 |    22159360 |    23060480 |           | 22 MB
 _timescaledb_internal | _hyper_2_389_chunk |      712704 |      196608 |    22380544 |    23289856 |           | 22 MB
 _timescaledb_internal | _hyper_2_407_chunk |      737280 |      204800 |    22814720 |    23756800 |           | 23 MB
 _timescaledb_internal | _hyper_2_419_chunk |      786432 |      212992 |    25272320 |    26271744 |           | 25 MB
 _timescaledb_internal | _hyper_2_597_chunk |     9584640 |     2449408 |   326180864 |   338214912 |           | 323 MB
 _timescaledb_internal | _hyper_2_605_chunk |     9404416 |     2441216 |   324444160 |   336289792 |           | 321 MB
 _timescaledb_internal | _hyper_2_613_chunk |     9551872 |     2449408 |   324853760 |   336855040 |           | 321 MB
 _timescaledb_internal | _hyper_2_491_chunk |     5414912 |      827392 |    92749824 |    98992128 |           | 94 MB
 _timescaledb_internal | _hyper_2_499_chunk |     7421952 |     1458176 |   167665664 |   176545792 |           | 168 MB
 _timescaledb_internal | _hyper_2_539_chunk |     9732096 |     2449408 |   325615616 |   337797120 |           | 322 MB
 _timescaledb_internal | _hyper_2_549_chunk |     8691712 |     1409024 |   172785664 |   182886400 |           | 174 MB
 _timescaledb_internal | _hyper_2_629_chunk |  2814263296 |  2704171008 |           0 |  5518434304 |           | 5263 MB
 _timescaledb_internal | _hyper_2_396_chunk |      712704 |      196608 |    22315008 |    23224320 |           | 22 MB
 _timescaledb_internal | _hyper_2_402_chunk |      729088 |      196608 |    22298624 |    23224320 |           | 22 MB
 _timescaledb_internal | _hyper_2_621_chunk |     9535488 |     2449408 |   325279744 |   337264640 |           | 322 MB
 _timescaledb_internal | _hyper_2_481_chunk |     3153920 |      753664 |    91643904 |    95551488 |           | 91 MB
 _timescaledb_internal | _hyper_2_425_chunk |      827392 |      221184 |    26394624 |    27443200 |           | 26 MB
 _timescaledb_internal | _hyper_2_431_chunk |      933888 |      229376 |    26640384 |    27803648 |           | 27 MB
 _timescaledb_internal | _hyper_2_438_chunk |      819200 |      229376 |    26853376 |    27901952 |           | 27 MB
 _timescaledb_internal | _hyper_2_445_chunk |      827392 |      229376 |    26828800 |    27885568 |           | 27 MB
 _timescaledb_internal | _hyper_2_566_chunk |     8208384 |     1368064 |   161792000 |   171368448 |           | 163 MB
 _timescaledb_internal | _hyper_2_637_chunk |  2813165568 |  2707677184 |           0 |  5520842752 |           | 5265 MB
 _timescaledb_internal | _hyper_2_573_chunk |      942080 |      245760 |    27025408 |    28213248 |           | 27 MB
 _timescaledb_internal | _hyper_2_645_chunk |  2830770176 |  2740445184 |           0 |  5571215360 |           | 5313 MB
 _timescaledb_internal | _hyper_2_582_chunk |      942080 |      245760 |    27303936 |    28491776 |           | 27 MB
 _timescaledb_internal | _hyper_2_653_chunk |  2859343872 |  2772500480 |           0 |  5631844352 |           | 5371 MB
 _timescaledb_internal | _hyper_2_452_chunk |     2146304 |      401408 |    38699008 |    41246720 |           | 39 MB
 _timescaledb_internal | _hyper_2_589_chunk |     8880128 |     1425408 |   171728896 |   182034432 |           | 174 MB
 _timescaledb_internal | _hyper_2_461_chunk |     2138112 |      532480 |    64856064 |    67526656 |           | 64 MB
 _timescaledb_internal | _hyper_2_467_chunk |     2359296 |      565248 |    66338816 |    69263360 |           | 66 MB
 _timescaledb_internal | _hyper_2_506_chunk |     8282112 |     2048000 |   266133504 |   276463616 |           | 264 MB
 _timescaledb_internal | _hyper_2_514_chunk |     8404992 |     2129920 |   280592384 |   291127296 |           | 278 MB
 _timescaledb_internal | _hyper_2_531_chunk |     9445376 |     2310144 |   300531712 |   312287232 |           | 298 MB
 _timescaledb_internal | _hyper_2_557_chunk |     8216576 |     1400832 |    94953472 |   104570880 |           | 100 MB
 _timescaledb_internal | _hyper_2_523_chunk |     8536064 |     2146304 |   283303936 |   293986304 |           | 280 MB
(91 rows)

``` 

Thanks for your support!

**Comment by FireHelmet** [comment]
Hello @akuzm ,

What should I do ?

Thank you for your support !

**Comment by FireHelmet** [comment]
Hello @akuzm and team,

Any news ?

Thanks !

**Comment by shhnwz** [comment]
Hi @FireHelmet ,

Thanks for being patient. This situation usually happens if in-progress compression errors out for some reason. This is a known issue and we have fixed it in the upcoming patch 2.11. 
As a workaround, we need to run VACUUM manually for bigger chunks (may be >500MB).

**Comment by FireHelmet** [comment]
Hello @shhnwz  and @akuzm 

Thank you for your support, may I ask you to tag in this thread which bugfix has solved the issue from the 2.11 ?

https://github.com/timescale/timescaledb/milestone/69?closed=1

Thank you !

**Comment by jnidzwetzki** [comment]
Hello @FireHelmet,

The [2.11.0 release](https://github.com/timescale/timescaledb/blob/main/CHANGELOG.md#2110-2023-05-12) of TimescaleDB contains the following patch that should solve the problem you mentioned:


> #5510 Propagate vacuum/analyze to compressed chunks

Starting with version 2.11.0, a `VACCUM` on the hypertable is now propagated to the compressed chunks. If you run a `VACCUM` on the hypertable, the space should be released. Please check whether version 2.11.0 completely solves the problem you reported or whether there is another issue on your system.
